package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tokenscript/token-negotiator",
-  "version": "1.0.18-alpha",
+  "version": "1.0.19-alpha-wd",
```

Comment:
Can we remove the -wd from the version. Let's move to using Semantic versioning.

https://docs.npmjs.com/about-semantic-versioning (applying these rules to future updates)

Reply:
Yep this was only so Feng can build the brand connector demo.